### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by ioNclash**

### CODE REVIEW

File: General.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+def sumEvenNumbers(arr):
+    '''accepts an array of integers and returns the sum of all the even numbers in the array
+        ARGS - arr: array of integers
+        RETURNS- sum of all even numbers: int'''
+    return sum(filter(lambda x: x%2 == 0,arr))
```
Comment (Goran-n) [inline General.py:5]: This is a really good implementation. Well done for using docstring

File: General.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+def sumEvenNumbers(arr):
+    '''accepts an array of integers and returns the sum of all the even numbers in the array
+        ARGS - arr: array of integers
+        RETURNS- sum of all even numbers: int'''
+    return sum(filter(lambda x: x%2 == 0,arr))
+
+def commonElements(arr1,arr2):
+    '''function that accepts two arrays and returns a new array that contains only the elements that are common between the two arrays'''
+    return list(set(arr1).intersection(set(arr2)))
```
Comment (Goran-n) [inline General.py:9]: Well done for using set intersection

File: General.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    '''accepts an array of integers and returns the sum of all the even numbers in the array
+        ARGS - arr: array of integers
+        RETURNS- sum of all even numbers: int'''
+    return sum(filter(lambda x: x%2 == 0,arr))
+
+def commonElements(arr1,arr2):
+    '''function that accepts two arrays and returns a new array that contains only the elements that are common between the two arrays'''
+    return list(set(arr1).intersection(set(arr2)))
+
+def secondSmallestInArray(arr):
```
Comment (Goran-n) [inline General.py:11]: Perfect implementation

File: General.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    return(arr[1])
+
+def palindromeChecker(string):
+    '''accepts a string and returns true if the string is a palindrome, false otherwise.'''
+    if string == string[::-1]:return True
+    return False
+
+def anagramChecker(str1,str2):
+    '''accepts two strings and returns true if they are anagrams of each other, false otherwise'''
+    if len(str1) == len(str2):
```
Comment (Goran-n) [inline General.py:23]: You don't need to loop through the strings to form a list as they are already a list in python

File: General.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        arr1,arr2 = [],[]
+        for char in str1:
+            arr1.append(char)
+        for char in str2:
+            arr2.append(char)
```
Comment (Goran-n) [inline General.py:28]: Good implementation

File: General.py (hunk 3 above)
Comment (jake-beardo) [reply General.py:11]: You would actually need to make it a set(arr) first before you sort as the array [1,1,2,3] we would want to return 2 not 1

**Pull request closed without merging**